### ISSUE TITLE
Add a free NTFS tool `Omi NTFS `

### ISSUE THREAD

**Issue by qizidog** [jaywcjlove/awesome-mac#983]
NOTE: Each discrete (stand-alone) request should be in its own issue.

**Put "Add ____" as Title**
 [x] Done

**Provide a link to the proposed addition.**
https://okaapps.com/product/1580856488

**Explain why it should be added**
Omi NTFS Disk Pro, Best Free Windows NTFS Disk Manager for Mac. Support Apple Silicon M1 Chip. Easily mount and unmount NTFS Disks on Mac. Support read, write, copy, edit files in Windows NTFS disk on Apple macOS, view all disk information. You can even repair and format the disk with one click.

**Additional context**
Compared to Paragon NTFS for Mac and Tuxera NTFS for Mac, it's **free! free! free!**. It can be downloaded from app store directly.

### Issue Checklist

Put an x in the boxes once you've completed each step. You can also fill these out after creating the issue. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before responding to the issue.

- [x] I have checked for other similar issues
- [x] I have explained why this change is important
- [x] I have added necessary documentation (if appropriate)

**Comment by christhomas** [comment]
In the past I've used paragon and tuxera and the annoying thing about them is that I need to downgrade my computer security by going into safe mode and disabling things. Do you remember that also? With this tool, do you need to do this also? Cause thats actually why I won't use paragon anymore. Cause I'm not going to downgrade my security. It's not worth it. So I don't want to use it anymore. Does this also require me to downgrade the security?

**Comment by christhomas** [comment]
Ah, that’s a shame because it’s a deal breaker for me. Thanks for letting me know!

**Comment by GitTheHellOutaHere** [comment]
I assume christhomas was referring to Gatekeeper? I always allow apps from anywhere, so that's not a problem for me. The installer didn't instruct me to disable anything, so I assume it recognized Gatekeeper was already disabled. **However,  the plug-in is not detected after rebooting.** 

When I open the installer it asks if I want to install again - doesn't say it's already installed. **com.omi.ntfs.HelperTool.plist** is in the LaunchDaemons folder, so what's the prob?

I've now wasted a bunch of time trying to figure out how to fix this. The USB drive I use with my tv appears in Omni's sidebar and can _supposedly_ be repaired, gray files fixed etc., but those buttons can't fix the missing plug-in issue.

I've now wasted a bunch of time trying to figure out how to make this work. SInce the plug-in isn't recognized, I can not add files to the drive, and when I select a file on the drive it can't be trashed. However I _can_ copy files on the drive to another volume.  I never had these issues with Paragon.

NTFS Disk by Omi 1.1.4
xNTFS Helper Installer 1.0.5

**Comment by Charlotte1959** [comment]
Approved

**Comment by christhomas** [comment]
@GitTheHellOutaHere no, I'm referring to these programs requiring me to go into my SIP security settings and downgrading my system integrity protection so it can install its driver and this is a problem for me because the last time I did this, a day later my system crashed out and I almost lost the entire contents of the disk that fortunately I was able to save because in the recovery the terminal had access to SSH that I could copy the drive contents to back them up.

The link: https://okaapps.com/blog/623abf19bbd93f236a277ed3 

explains the process that I'm referring to.

**Comment by Charlotte1959** [comment]
Approved
